### DISCUSSION TITLE
Aqara mini switch is unreliable.

### DISCUSSION THREAD

**Discussion by braucktoon** [Koenkk/zigbee2mqtt#12675] [Devices]
Hello. I have a pretty simple network. Really just testing some devices right now. I was using ZHA but it would just stop out of nowhere and I had to unplug and plug in the coordinator (Sonoff usb zigbee 3.0 with latest firmware) to fix it.

I am trying Z2M in hopes for some stability but I'm facing an issue that I can't quite work out. 

I have a Aqara mini switch and when I create an automation based on state trigger, the switch is not 100% reliable. It can take several clicks of the button to get the automation to run successfully. But once it actually runs it does not fail. It's almost as if the switch is going to sleep and when I click the button it doesn't wake up until I click it a few times. For the record I never had this issue using ZHA. Switches were very reliable under that configuration. 

I've made sure that I have legacy Mode set the false for the switch itself Z2M. I've also made sure that home assistant legacy triggers are enabled. 

I've tried using a blueprint (https://community.home-assistant.io/t/aqara-buttons-all-in-one-zigbee2mqtt/268717) and I tried creating my own automation but they both suffer from the same issue. 

My switch is connected to my IKEA outlet router which is connected to my coordinator and the connection seems very strong. 

I've also been using the lights in the automation for a long time and they are rocksolid through Home Assistant so I'm pretty sure it's something either with Zigbee2MQTT. 

Any help as appreciated. Thank you.

**Comment by gofort** [comment]
Same issue as here: https://github.com/Koenkk/zigbee2mqtt/issues/15987
